indico/modules/groups/templates/group_details.html

```diff
@@ -15,11 +15,14 @@
             {% trans %}Delete group{% endtrans %}
         </button>
     {% endif %}
+
     <h3>{% trans %}Group members{% endtrans %}</h3>
     <div id="group-members">
         {% if group.is_local %}
             {# Local groups usually don't have massive amounts of members so we load them immediately #}
             {{ group_members(group) }}
+        {% elif not group.supports_member_list %}
+            <strong>{% trans %}The data source of this group does not allow retrieving its member list.{% endtrans %}</strong>
```

Comment:
```suggestion
            {% call message_box('warning') %}
                {% trans %}The data source of this group does not allow retrieving its member list.{% endtrans %}
            {% endcall %}
```

Note: You also need to add `{% from 'message_box.html' import message_box %}` to the imports at the top for this to work.

Reply:
It's more intuitive indeed. I just applied the simple message style from `_group_members.html`.